README.md

```diff
@@ -57,6 +57,37 @@ for ($i = 0; $i < 3; $i++) {
 // 'Orlo Bergstrom'
 ```
 
+## Automated refactoring
+
+If you already used this library with its properties, they are now deprecated and needs to be replaced by their equivalent methods.
+
+You can use the provided [Rector](https://github.com/rectorphp/rector) config file to automate the work.
+
+Run
+
+```bash
+$ composer require --dev rector/rector
+```
+
+to install `rector/rector`.
+
+Run
+
+```bash
+vendor/bin/rector process src/ --config vendor/fakerphp/faker/rector-migrate.php
+```
+
+to run `rector/rector`.
+
+*Note:* do not forget to replace "src/" with the path to your source directory.
+
+Another way is to use it in your `rector.php` file:
+
+```php
+$faker = require 'vendor/fakerphp/faker/rector-migrate.php';
```

Comment:
This can use the import from Rector

```suggestion
$rectorConfig->import('vendor/fakerphp/faker/rector-migrate.php');
```

Reply:
Oh I didn't know about this 😄 

Thanks!